docs/SnapshotInstallation.md

```diff
@@ -0,0 +1,19 @@
+# Snapshot Installation Instructions
```

Comment:
For clarify, suggestion re-naming to "Plugin Source Build Installation"

Reply:
Done